airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/utils/dagster_helpers.py

```diff
@@ -13,28 +14,15 @@ def output_dataframe(result_df: pd.DataFrame) -> Output[pd.DataFrame]:
     return Output(result_df, metadata={"count": len(result_df), "preview": MetadataValue.md(result_df.to_markdown())})
 
 
-def deserialize_composite_etags_cursor(etag_cursors: Optional[str]) -> List[str]:
-    """Deserialize a cursor string into a list of etags.
+def string_array_to_cursor(strings: List[str]) -> str:
```

Comment:
```suggestion
def hash_list_of_string(strings: List[str]) -> str:
```

I would find it more readable to rename it this way. I orinally thought cursor was a specific type.